MLS.Agent.Tests/ApiViaHttpTests.cs

```diff
@@ -768,10 +768,24 @@ public static void Main()
         {
             Clock.Reset();
 
-            var workspace =
-                workspaceType == "script"
-                    ? Workspace.FromSource(code, "script")
-                    : Workspace.FromSource(code, (await Create.ConsoleWorkspaceCopy()).Name);
+            Workspace workspace = null;
+            if (workspaceType == "script")
+            {
+                workspace = Workspace.FromSource(code, "script");
+            }
+            else
+            {
+                var package = Create.EmptyWorkspace();
+                var build = await Create.NewPackage(package.Name, package.Directory, packageBuilder =>
+                {
+                    packageBuilder.CreateUsingDotnet("console");
+                    packageBuilder.TrySetLanguageVersion("8.0");
+                    packageBuilder.AddPackageReference("Newtonsoft.Json");
+                });
```

Comment:
Please move all package building into a utility method somewhere rather than repeating it from test to test.